components/newsroom-header/template-parts/index.php

```diff
@@ -11,7 +11,8 @@
 namespace Civil_First_Fleet;
 
 // Get this instance.
-$ai_component = ai_get_var( 'component' );
+$ai_component           = ai_get_var( 'component' );
+$disable_civil_branding = wp_validate_boolean( get_option( 'newsroom-settings' )['branding']['civil_branding']['disable_header_cta'] ?? false );
```

Comment:
I know that this code fits nicely on a single line, but attempting to accessing array elements directly from a function call is not ideal. This is not a hard blocker, but I would suggest either creating a helper function to get the value like this 

```php
function disable_civil_branding() {
    $settings = get_option( 'newsroom-settings' );

    return ! empty( $settings['branding']['civil_branding']['disable_header_cta'] );
}
```
or just breaking out the code inline to multiple parts. Since `get_option` can return a value of `false` I believe this will throw a PHP notice when there is no value set (which I understand is not likely, but still something to cover).

Reply:
> I believe this will throw a PHP notice when there is no value set

Not quite,
```
<?php
$get_option = false;

$result = $get_option['one']['two'] ?? 'default';

echo $result; // prints `default`.
```

When a new site gets created, get_option here would absolutely return false, but the null coalescing operator takes care of it.

Reply:
It's the only place it's used, so I'm hesitant to prematurely refactor and add clutter elsewhere. If you have some additional info or docs about why it's not good I'm happy to take a look and be convinced otherwise :)

Reply:
This is really just about readability, and I see this code all being on a single line as less readable than having it broken out (or in a helper function). It is not an explicit standard, but just a coding style that I personally believe is more readable.

Reply:
Fair, and I have the opposite opinion. I like having it on one line, in one place, where I can see and understand exactly what's happening w/o needing to find an abstraction.

20 ways to dress a monkey right?

Reply:
Well, adding a helper function for this is not abstractions, it is just DRY :)

Reply:
This is the only place this flag is used, moving into a standalone function isn't any more dry, it's the same code, just in a different place. If I was calling this in a few places, I'd totally agree.

Reply:
It seems to be used twice, for the CTA and the footer. So having a helper function that accepts a string for the setting you want to check might be valuable.